BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -33,9 +32,10 @@ public PlatformImplementation()
             {
                 this.environmentVariables = Environment.GetEnvironmentVariables();
             }
-            catch (SecurityException e)
+            catch (Exception e)
             {
                 CoreEventSource.Log.FailedToLoadEnvironmentVariables(e.ToString());
+                throw;
```

Comment:
Catching the generic exception is correct :)

But we don't want these methods to throw exceptions. The EventSource log will create internal traces that we can monitor.

@cijothomas what is the correct default value to return in this instance?